airbyte-integrations/connectors/source-railz/source_railz/railz.yaml

```diff
@@ -0,0 +1,362 @@
+version: "0.1.0"
```

Comment:
could you rename this file to `manifest.yaml` and update `source.py` accordingly? see https://github.com/airbytehq/airbyte/pull/21771 for context